src/app/Models/UploadFile.ts

```diff
@@ -2,7 +2,7 @@ import { AiurProtocal } from './AiurProtocal';
 
 export class UploadFile extends AiurProtocal {
```

Comment:
It seems that this class already not exists anymore.

Reply:
?